Automattic/wp-calypso
# Launchpad: Enable the design-first flow to use the new task definition

Related to https://github.com/Automattic/wp-calypso/issues/86647

## Proposed Changes
Enable the design-first flow to use the new task definition parser for testing purposes, since it is using already migrated tasks. 


'design_completed',
'setup_blog',
'domain_upsell',
'plan_completed',
'first_post_published',
'blog_launched',
				
All tasks related to this flow were already migrated, so this PR is just enabling the feature flag.


## Testing Instructions
* Start the flow ` design-first`  (setup/design-first)
* Follow the flow until arrive on the launchpad page (setup/start-writing/launchpad)
* Enable the feature flags via url
* Check if  there are messages on devtools to all tasks above
* Use the launchpad (Set values, Edit values, Check if the values are marked when completed)
* Finish the flow.

Requires the feature flags:
* "launchpad/new-task-definition-parser"
* "launchpad/new-task-definition-parser/design-first"

The flags can be enabled appending  the `?flags=` flag on the url (Please always check if the flags= param is still on the url when the system redirect back the user from some "no launchpad" page ).

## Pre-merge Checklist
